File: mds-demo-assignment/tests/q1.py
OK_FORMAT = True

test = {
    "name": "q1",
    "points": 1,
    "suites": [{
        "cases": [
            {"code": ">>> isinstance(mean_wait, float)\nTrue", "hidden": False},
            {"code": ">>> round(float(mean_wait), 2) == 3.71\nTrue", "hidden": False},
        ],
        "scored": True,
        "type": "doctest",
    }],
}

File: mds-demo-assignment/release/lab0b/student/tests/q1.R
test = list(
  name = "q1",
  cases = list(
    ottr::TestCase$new(
      hidden = FALSE,
      name = NA,
      points = 1.0,
      code = {
        testthat::expect_true(!is.null(r_courses))
        testthat::expect_equal(as.numeric(r_courses), 3)
      }
    )
  )
)